tests/unit/test_client_options.py

```diff
@@ -28,11 +28,18 @@ def get_client_encrypted_cert():
 def test_constructor():
 
     options = client_options.ClientOptions(
-        api_endpoint="foo.googleapis.com", client_cert_source=get_client_cert
+        api_endpoint="foo.googleapis.com",
+        client_cert_source=get_client_cert,
+        quota_project="quote-proj",
+        credentials_file="path/to/credentials.json",
+        scopes=["..", "..."]
```

Comment:
Sorry, I was being lazy when I opened that issue and didn't find proper scopes. 😅 

Could you choose one or two from [this list](https://developers.google.com/identity/protocols/oauth2/scopes) and use them for these tests?

Reply:
Done